### ISSUE TITLE
Integrate 1Password SDKs

### ISSUE THREAD

**Issue by carey404** [benvp/insomnia-plugin-op#2]
Hey there,

Curious if you're open to adding support for the new 1Password SDKs to this Insomnia plugin. The new SDKs allow direct access to 1Password secrets without needing to install the CLI, streamlining the integration process and reducing dependencies.

**Benefits**:
- Load secrets directly from 1Password using SDKs in Go, JS, or Python
- Retrieve, create, edit, and delete items within 1Password vaults.
- Use 1Password Service Accounts to authenticate. 

Feel free to reach out to discuss in more detail. Thanks!

**Comment by benvp** [comment]
Absolutely. I see it's still in beta phase, so I'd prefer to wait for it to come stable. As soon as it's stable it should be easy enough to integrate. 

Thanks for letting me know about it, I didn't notice they released a new sdk.
